doc/en/multitest.rst

```diff
@@ -276,6 +276,40 @@ and overriding ``get_output`` method.
 An example implementation of custom test lister can be seen
 :ref:`here <example_multitest_listing_custom>`.
 
+If the lister provide, name and description it can be used to extend the ``--info`` commandline
+parameter and the lister can be selected from the commandline.
+
+The class should provide:
+
+* it's name either setting the :py:attr:`NAME <testplan.testing.listing.BaseLister.NAME>` or override the
+  :py:meth:`name() <testplan.testing.listing.BaseLister.name>` method. This should be an Enum name like ``NAME_FULL``.
+  The name will be used to derive the commandline param which is the kebab-case version of the name.
```

Comment:
Typo: kebab?

Reply:
do you mean 'kebab version' instead of 'kebab-case version'?